lms/services/upsert.py

```diff
@@ -16,6 +17,8 @@ def bulk_upsert(
     """
     Create or update the specified values in a table.
 
+    Returns a lazy query of the affected `model_class` rows.
```

Comment:
Might as well but this in a `:return:` at the bottom of the docstring, since we're using `:param:`